### DISCUSSION TITLE
mklink.bat文件在哪

### DISCUSSION THREAD

**Discussion by DrownedFish21** [LorisYounger/VPet#132] [General]
我在桌面端部署，缺少模组Core,无法启动桌宠，但我找不到mklink.bat文件来让mod文件链接到生成位置。请问mklink.bat在哪

**Comment by DrownedFish21** [comment]
已解决。我从Steam上下载到本地，然后将其中的mod文件夹拷贝到代码所指定的位置。

**Comment by LorisYounger** [comment]
在 VPet-Simulator.Windows
